web/css/style.css

```diff
@@ -0,0 +1,239 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap');
+@import "font.css";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto;
+  margin: 0 36px;
+}
+
+.main-title {
+  height: 40px;
+  font-size: 36px;
+  text-align: center;
+  margin: 0;
+}
+
+.second-title {
+  height: 30px;
+  font-size: 24px;
+  text-align: center;
+  margin: 0;
+}
+
+.section-title,
+.section-subtitle {
+  height: 30px;
+  margin: 0;
+  color: #808080;
+}
+
+.section-title {
+  font-size: 22px;
+}
+
+.section-subtitle {
+  font-size: 22px;
+}
+
+grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 10px;
+}
+
+grid > c {
+  /* overflow: hidden; */
+}
+
+grid > c[span="row"] {
+  /* grid-column-start: 1;
+  grid-column-end: -1; */
+  grid-column: 1 / -1;
+}
+
+c[height="30"] {
+  height: 30px;
+}
+
+.box {
+  background-color: #fafafa;
+  padding: 20px;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.25);
+}
+
+.description {
+  margin: 0;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 30px;
+  color: #808080;
+}
+
+.bAtention {
```

Comment:
why did you use camel case?

Reply:
Done!
Now I understand camel case are for JS and not for CSS